__tests__/__system__/cli/list/job/__scripts__/command/command_list_job_noprefix.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+bright zos-ftp list jobs
```

Comment:
this probably should be `zowe zos-ftp list jobs` - the `bright` high level command name still works, but it generally not used any longer.